awkward/array/jagged.py

```diff
@@ -104,6 +104,18 @@ def parents2startsstops(cls, parents, length=None):
 
         return starts, starts + counts
 
+    @classmethod
```

Comment:
This implementation is not general enough to be a public-facing function. If, for instance, the stops are out of order, it should be `stops.max()`, not `stops[-1]`. However, I don't think you should generalize it; it's general enough for your case (`starts` and `stops` made from `counts`). I think you should hide it with an underscore or absorb it into your public function.

Motivation behind that request: I don't see how it would be useful in general. I see why your new function needs it, but not beyond that.